src/theme/typography.scss

```diff
@@ -1,7 +1,7 @@
-$font-sans-serif: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+$font-sans-serif: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto",
```

Comment:
not sure why this is automatically making it lower case.  I didn't manually make this change.

Reply:
That's the linter. The matching is case insensitive, so it is fine.